### ISSUE TITLE
error running northwind.sql in MySQL Workbench

### ISSUE THREAD

**Issue by dataSQL** [dalers/mywind#2]
Please, how would I run the script in MySQL Workbench, it keeps returning an error.
FILE 
RUN SQL SCRIPT
northwind.sql 

results in 

error calling Python module function SQLIDEUtils.runSQLScript

any suggestions greatly appreciated.

Thank you.

**Comment by dalers** [comment]
I suspect there a problem with your MySQL Workbench installation, although it is possible MySQL Workbench could be responding very poorly to an error encountered while processing the northwind.sql file. I suggest you first test your MySQL Workbench install using a very simple SQL script (such as from an on-line tutorial, or a very cut back version of northwind.sql. Once this works, then try northwind.sql again.

**Comment by dataSQL** [comment]
Hello, with all respect...
I was actually able to run another SQL script I found and it worked perfect and I was able to run the script to create the tables in a new schema on AWS.
here is the download link for comparisons -
[ http://dev.wavemaker.com/forums/sites/default/files/files/northwind.sql_.zip
](url)

**Comment by dalers** [comment]
Thanks @TahorSuiJurisBenYAH for the information and reference. Be aware though that this database you referenced may be considerably different than the schema here; the "Northwind Traders" database provided with Microsoft Access evolved considerably over time and also as Access evolved. I have also seen implementations that didn't use foreign keys.

I just downloaded the current northwind.sql file from the MyWind project and it was read without error by  [MariaDB 10.0.27](https://mariadb.org/). I don't have a MySQL server immediately available, but I would expect the same results with MySQL (MySQL and MariaDB should be fully compatible so far as MyWind is concerned). I used [Adminer 4.2.5](https://www.adminer.org/) (a web-client supporting a variety of DBMS including MySQL/MariaDB) to import the file into MariaDB.

northwind.sql was generated by MySQL Workbench and, besides expecting it to be somewhat generic, I would expect MySQL Workbench to be able to read files it has written, I would be interested in working together to resolve the problem should you want to pursue this further. Did you try using the model (northwind.mwb) in MySQL Workbench instead of the SQL file? Were you also using AWS when you encountered your original error? Did you test on any other systems, such as MySQL/MariaDB?

**Comment by dalers** [comment]
will leave open a while longer....

**Comment by dataSQL** [comment]
Thank you.  No harm intended, just a desire to improve my knowledge.
Jon

**Comment by dalers** [comment]
I reviewed README.md and suspect my comment earlier regarding the origin of the SQL file is incorrect, or at best a guess. The SQL is possibly edited output from either BullZip or MySQL Workbench, or could even be the edited export or dump from MySQL.

I used the Northwinds schema (i.e. MyWind) as the basis of a simple financial schema for my [Maestro project](https://github.com/dalers/maestro), a simple ERP for engineering shops doing project-based custom design and low-volume traceable-type manufacturing. The parts management schema in [Maestro](https://github.com/dalers/maestro) came from a now-abandonned commercial PC app called [Parts&Vendors](http://www.trilogydesign.com/). My goal with MyWind was simply to wrap up what I had learned about the Northwinds schema, I would have liked to have left the project in better shape but unfortunately priorities change. However, I still believe a basic ERP schema that could be hosted anywhere (e.g. AWS) would be useful so who knows where the future lies....

Thanks for stopping by. I'm sorry it didn't work out for you but it was nice chatting.
